### ISSUE TITLE
Friend List

### ISSUE THREAD

**Issue by chpapa** [SkygearIO/features#92]
# Description

Friend List, FB import Friends

# Portal Design

**Remove this section if the feature have no Portal**

## Portal Features

**Put Portal Feature List here**

# API Design

**Remove this section if the feature have no API**

## Scenario

**Put list of Scenario how developers will use this API**

# Open Questions

**Put a list of open questions here before a complete design / specification is decided**

# Related Issues

- Server Issues
- Client Issues
- Guides Issues

# Progress Tracker

- [ ] Specification Design Approval
- [ ] Write (code + tests + API docs) then get them merged. **Put All PR there**
  - [ ] Code
  - [ ] Minimal tests
  - [ ] Minimal API Docs
- [ ] Guides. **Put All PR there**
- [ ] Release
  - [ ] Update release notes
  - [ ] Release

# Advice

- Specification Design Approval
  - Once you get LGTM from another Skygear Core Team Member, you can check this checkbox. And apply the "workflow/design-complete" label.
- Coding
  - Use as many PRs as you need.  Write tests in the same or different PRs, as is convenient for you.
  - API doc should goes in the same PR with the code.
  - As each PR is merged, add a comment to this issue referencing the PRs.
  - When you are done with the code, apply the "workflow/code-complete" label.
- Guides
  - Write or modify guides and get them merged in https://github.com/skygeario/guides
  - When the PR of guides is merged, check this checkbox and apply the "workflow/guides-complete" label.

**Comment by juicycleff** [comment]
Do you accept contributors. I can contribute on the React Native, JavaScript and Android, Payment Plugin part. Its awesome what you guys are doing.

**Comment by chpapa** [comment]
@juicycleff definitely! Feel free to propose the design of how the SDK will be used and we would love to assist you with put together a good spec for implementation.
